dc/council/code/titles/42/sections/42-818.02.xml

```diff
@@ -70,57 +70,64 @@
+    <text>If a holder of a promissory note secured by a deed of trust fails to execute, acknowledge, and deliver, or cause to be executed, acknowledged, and delivered, the documents, instruments, or affidavits required to release the deed of trust, in whole or in part, within the time, and in the manner, required by paragraph (1) or (2) of this subsection, and if the holder’s failure continues for more than 30 days after the holder receives a written request therefor from the person entitled to the release or such person’s agent, then holder shall pay to the person entitled to the release a penalty in the amount of $50 per day, shall be liable to such person for all actual and consequential damages caused by the holder’s failure timely to deliver or record the full or partial release, and shall pay or reimburse such person for all costs and expenses, including reasonable attorneys fees and disbursements, relating to or arising out of the enforcement of such person’s rights under this section. The penalty of $50 per day shall be payable for the period beginning on and including the 31st day after the holder receives a written request for the release to, but not including, the day on which the holder delivers the executed and acknowledged documents, instruments or affidavits required to release the deed of trust.</text>
+    <text>For purposes of this subsection, (i) a payment in the form of an electronic transfer of immediately available funds to an account in a commercial bank, a savings bank, a savings and loan association, a credit union or a similar financial institution shall be deemed to be made when the financial institution confirms receipt of the funds to the owners of the account, (ii) a payment in the form of a check issued or certified by a national or state bank shall be deemed to be made upon receipt of the check, and (iii) payment in the form of a check that is not issued or certified by a national or state bank shall be deemed to be made on the first day on which the holder receives the proceeds of collection of such check in immediately available funds.</text>
+    <text>If a deed of trust is released, or deemed released, as a lien on all of the real property described therein, the release of the deed of trust shall be deemed automatically to release any ancillary security instrument that secures the same promissory note secured by the deed of trust. This provision shall not apply if the document recorded among the land records expressly states that the release of the deed of trust shall not release the ancillary security instrument.</text>
+    <text>
+      <center>
+        <strong>
+          FORM OF RELEASE AFFIDAVIT FOR LOST, MISPLACED, OR DESTROYED PROMISSORY NOTE PER § 45-721(C)(1) [see now § 42-1101(3)(A)]:
+          <br/>
+          KNOW ALL MEN BY THESE PRESENTS:
```

Comment:
Compare with line 117 of this file. Not sure what change, if any, should be made.